### PR TITLE
[DOCFIX] Update bin/alluxio usage format

### PR DESCRIPTION
**Pull request by Xenorith**
besides the User-CLI.md doc, update other doc files that refer to `bin/alluxio` commands
- remove docs on path config
- remove starting/stopping job master/worker from contributor docs

### CODE REVIEW

File: docs/en/api/REST-API.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ can use in-memory streams, the REST API decouples the stream creation and access
 `create` and `open` REST API methods and the `streams` resource endpoints for details).
 
 The HTTP proxy is a standalone server that can be started using
-`${ALLUXIO_HOME}/bin/alluxio-start.sh proxy` and stopped using `${ALLUXIO_HOME}/bin/alluxio-stop.sh
-proxy`. By default, the REST API is available on port 39999.
+`${ALLUXIO_HOME}/bin/alluxio process start proxy` and stopped using `${ALLUXIO_HOME}/bin/alluxio process stop proxy`.
```
Comment (Kai-Zhang) [inline docs/en/api/REST-API.md:24]: talked to @JiamingMai and i think the proxy is no longer support in 30x.

**Review by Kai-Zhang — APPROVED** [review]
Overall LGTM

File: docs/en/api/REST-API.md (hunk 1 above)
Comment (Xenorith) [reply docs/en/api/REST-API.md:24]: we still have the code for the process; is there a plan to remove it?
should this doc be completely removed/replaced? @LuQQiu i remember you mentioned the rest api was being replaced; any suggestions on how to handle the doc page?

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Pull request merged**